docusaurus/docs/iOS/05-ui-cookbook/15-long-press-to-focus.mdx

```diff
@@ -9,15 +9,45 @@ The StreamVideo SDK allows for interactive and intuitive user engagement during
 
 ## Overview
 
-- **Gesture Recognition**: Detect a long press on a participant's video feed.
 - **Focus Implementation**: Focus on the specified point on your video feed.
+- **Gesture Recognition**: Detect a long press or a tap on a participant's video feed.
 - **User Experience**: Intuitive interaction for a more immersive call experience.
 
 ## Implementing Long Press to Focus
 
+### Focusing on a point in the Video Feed
+
+In order to focus the camera on the desired we need to forward the request the WebRTC videoCapturer. Before we do that we need to create a call and then join it:
```

Comment:
"camera on the desired we need", seems like you missed something here. Probably "camera at the desired focus point..."?